src/data/osquery_metadata.json

```
@@ -1,5 +1,5 @@
 {
-  "current_version":"4.1.1",
+  "current_version":"4.1.2",
```

```suggestion
  "current_version":"4.1.1",
```

I think we should _not_ change this, and publish without